services/docker-compose.yml

```diff
@@ -20,17 +20,8 @@ services:
     environment:
       - KMS_WORKSPACE=/workspace
```

Comment:
shouldn't we expose the random port used by the jwt issuer?

Reply:
We should and we do, otherwise it wouldn't be possible to get tokens, if you look two lines down you'll see it's emitted into a file in the workspace which we can read